spec/system/candidate_interface/candidate_completes_the_satisfaction_survey_spec.rb

```diff
@@ -20,6 +20,50 @@
     when_i_choose_5
     and_click_continue
     then_i_see_the_ease_of_use_page
+
+    when_i_choose_1
+    and_click_continue
+    then_i_see_the_help_needed_page
+
+    when_i_choose_1
+    and_click_continue
+    then_i_see_the_organisation_page
+
+    when_i_choose_1
+    and_click_continue
+    then_i_see_the_consistency_page
+
+    when_i_choose_1
+    and_click_continue
+    then_i_see_the_adaptability_page
+
+    when_i_choose_1
+    and_click_continue
+    then_i_see_the_awkward_page
+
+    when_i_choose_1
+    and_click_continue
+    then_i_see_the_confidence_page
+
+    when_i_choose_1
+    and_click_continue
+    then_i_see_the_needed_additional_learning_page
+
+    when_i_choose_1
+    and_click_continue
+    then_i_see_the_improvements_page
+    #
```

Comment:
Do we need these comments?

Reply:
They're for the next PR

Reply:
Up to you if they are useful leave them I say - esp as they will be out or code after next PR.